### PR TITLE
remove div wrapper from context

### PR DESCRIPTION
**Pull request by strausr**

### CODE REVIEW

File: src/components/CloudinaryContext/CloudinaryContext.js [hunk 1]
```diff
@@ -20,6 +21,7 @@ class CloudinaryContext extends CloudinaryComponent {
 
   getChildContext() {
     let context = {};
+    let shouldReturnDiv = true;
```
Comment (asisayag2) [inline src/components/CloudinaryContext/CloudinaryContext.js]: @strausr , where are you using this?

File: src/components/CloudinaryContext/CloudinaryContext.js [hunk 2]
```diff
@@ -40,7 +42,10 @@ class CloudinaryContext extends CloudinaryComponent {
         allowedProps[currentProp] = this.props[currentProp];
         return allowedProps;
       }, {});
-    return <div {...nonCloudinaryProps}>{this.props.children}</div>;
+    const {includeOwnBody, ...props} = nonCloudinaryProps;
+    //{console.log(nonCloudinaryProps.includeonbody);}
```
Comment (asisayag2) [inline src/components/CloudinaryContext/CloudinaryContext.js]: @strausr , please remove comments

File: src/components/CloudinaryContext/CloudinaryContext.js [hunk 3]
```diff
@@ -40,7 +42,10 @@ class CloudinaryContext extends CloudinaryComponent {
...
+  
+  return nonCloudinaryProps.includeOwnBody ? this.props.children : <div {...props}>{this.props.children}</div>;
```
Comment (asisayag2) [inline src/components/CloudinaryContext/CloudinaryContext.js]: @strausr , in line 45 you extracted the value of nonCloudinaryProps.includeOwnBody into a variable, why not using it?

File: src/components/CloudinaryContext/CloudinaryContext.js [hunk 4]
```diff
@@ -53,8 +58,10 @@ CloudinaryContext.CLOUDINARY_PROPS = CloudinaryComponent.VALID_OPTIONS.reduce(
   {}
 );
 
-CloudinaryContext.propTypes = CloudinaryComponent.propTypes;
-CloudinaryContext.defaultProps = {};
+CloudinaryContext.propTypes = {...CloudinaryComponent.propTypes, includeOwnBody: PropTypes.bool};
+//CloudinaryContext.propTypes.includeonbody = PropTypes.bool;
```
Comment (asisayag2) [inline src/components/CloudinaryContext/CloudinaryContext.js]: @strausr , please remove comments

File: test/ContextTest.js [hunk 5]
```diff
@@ -19,6 +19,16 @@ describe('CloudinaryContext', () => {
     expect(img.instance().state.url).to.equal("http://res.cloudinary.com/demo/image/upload/sample");
   });
 
+   it("should render without div", function() {
```
Comment (asisayag2) [inline test/ContextTest.js]: @strausr, please also create a test to make sure that not sending isOwnBody or sending false, creates a 'div'

**Comment by asisayag2** [comment]
@strausr , @tocker , any idea why the tests are failing?

**Comment by tocker** [comment]
@asisayag2 the source x node 6 test fails because some api is not supported in node 6. I'll exclude this test from the Travis matrix.

**Pull request merged**